fixed_bed_model.py

```diff
@@ -0,0 +1,2630 @@
+import matplotlib.pyplot as plt
+from pyomo.environ import *
+from pyomo.dae import *
+
+import numpy as np
+from scipy.interpolate import interp2d
+import pandas as pd
+
```

Comment:
Add a comment block here listing the full references. You can then refer to them with shorthand, e.g., [Hughes et al., 2021] in the remainder of the comments.

Reply:
added reference list here